src/sentry/static/sentry/app/views/settings/projectDataPrivacy/projectDataPrivacyContent.tsx

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import {t} from 'app/locale';
+import SettingsPageHeader from 'app/views/settings/components/settingsPageHeader';
+import JsonForm from 'app/views/settings/components/forms/jsonForm';
+import Form from 'app/views/settings/components/forms/form';
+import {fields} from 'app/data/forms/projectGeneralSettings';
+import AsyncView from 'app/views/asyncView';
+import ProjectActions from 'app/actions/projectActions';
+import {Organization} from 'app/types';
+
+export type ProjectDataPrivacyContentProps = {
+  organization: Organization;
+  params: {
+    orgId: string;
+    projectId: string;
+  };
+};
+
+class ProjectDataPrivacyContent extends AsyncView<ProjectDataPrivacyContentProps> {
+  static contextTypes = {
+    organization: PropTypes.object.isRequired,
```

Comment:
Similar comment to this: https://github.com/getsentry/sentry/pull/17456#discussion_r388533350

Reply:
@billyvg I understand you, but they are not propTypes but contextTypes and if I remove them I will break some tests we have

Reply:
@billyvg I understand you and also wanted to remove the static contextTypes, but..please see:

![image](https://user-images.githubusercontent.com/29228205/76067993-a1a38900-5f90-11ea-9763-8c5e2b7b63cc.png)